### PR TITLE
Correct configuration of QuantumESPRESSO for 6.7/6.8/7.0/7.1/7.2/7.3

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)
- [x] https://github.com/easybuilders/easybuild-easyblocks/pull/3241
- [x] https://github.com/easybuilders/easybuild-easyblocks/pull/3338

### CODE REVIEW

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3241"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3241"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3241"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3241"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3241"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3241 QuantumESPRESSO-7.0-foss-2021b.eb QuantumESPRESSO-7.1-intel-2022a.eb"

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-foss-2022b.eb [hunk 1]
```diff
@@ -45,6 +45,9 @@ dependencies = [
 # which depends on qe source
 buildopts = 'all gwl xspectra couple epw gipaw w90'
 
+# building gipaw requires FoX
```
Comment (Micket) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-foss-2022b.eb:48]:
```suggestion
```

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-foss-2023a.eb [hunk 2]
```diff
@@ -46,6 +46,9 @@ dependencies = [
 # which depends on qe source
 buildopts = "all gwl xspectra couple epw gipaw w90"
 
+# building gipaw requires FoX
```
Comment (Micket) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-foss-2023a.eb:49]:
```suggestion
```

File: easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-intel-2022b.eb [hunk 3]
```diff
@@ -45,6 +47,9 @@ dependencies = [
 # which depends on qe source
 buildopts = 'all gwl xspectra couple epw gipaw w90'
 
+# building gipaw requires FoX
```
Comment (Micket) [inline easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-7.2-intel-2022b.eb:50]:
```suggestion
```

**Review by Micket — CHANGES_REQUESTED** [review]
Based on https://github.com/easybuilders/easybuild-easyblocks/pull/3258

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3241 QuantumESPRESSO-7.2-foss-2022b.eb QuantumESPRESSO-7.2-intel-2022b.ebQuantumESPRESSO-7.2-foss-2023a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.3-foss-2023a.eb QuantumESPRESSO-7.3-intel-2023a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.2-foss-2023a.eb QuantumESPRESSO-7.2-intel-2022b.eb QuantumESPRESSO-7.2-foss-2022b.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.3-foss-2023a.eb QuantumESPRESSO-7.3-intel-2023a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.1-foss-2022a.eb QuantumESPRESSO-7.1-intel-2022a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.0-foss-2021b.eb QuantumESPRESSO-7.0-intel-2021b.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.0-foss-2021b.eb QuantumESPRESSO-7.0-intel-2021b.eb QuantumESPRESSO-6.7-foss-2021a.eb QuantumESPRESSO-6.7-intel-2021a.eb QuantumESPRESSO-6.8-foss-2021a.eb QuantumESPRESSO-6.8-intel-2021a.eb QuantumESPRESSO-6.8-foss-2021b.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.3-foss-2023a.eb QuantumESPRESSO-7.3-intel-2023a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.2-foss-2023a.eb QuantumESPRESSO-7.2-intel-2022b.eb QuantumESPRESSO-7.2-foss-2022b.eb"

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.0-intel-2021b.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3306 QuantumESPRESSO-7.0-intel-2021b.eb

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3338 QuantumESPRESSO-7.3-foss-2023a.eb"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3338 QuantumESPRESSO-7.0-intel-2021b.eb"

**Comment by ocaisa** [comment]
@boegel This is now succeeding for all easyconfigs in the PR _except_ `QuantumESPRESSO-7.0-intel-2021b.eb` which seems to be totally broken, I have no fix for that so I am just going to remove it from the PR and pretend I didn't notice

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
There are cases where there are a significant number of test failures in old QE easyconfigs. The test suite was never run in the past so these were not caught. With the merge of https://github.com/easybuilders/easybuild-easyblocks/pull/3338 I've explicitly skipped the test step for these cases.

**Comment by ocaisa** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3338"

**Comment by ocaisa** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3338"

**Review by jfgrimm — APPROVED** [review]

**Pull request merged**